tests/visco_plastic_phases_viscosity_max_min.prm

```diff
@@ -0,0 +1,125 @@
+set End time                               = 0
+set Use years in output instead of seconds = true
+set Nonlinear solver scheme                = single Advection, iterated Stokes
+set Max nonlinear iterations               = 1
+set Output directory                       = visco_plastic
+set Timing output frequency                = 1
+
+# Model geometry (100x100 km, 10 km spacing)
+subsection Geometry model
+  set Model name = box
+  subsection Box
+    set X repetitions = 10
+    set Y repetitions = 10
+    set X extent      = 100e3
+    set Y extent      = 100e3
+  end
+end
+
+# Mesh refinement specifications
+subsection Mesh refinement
+  set Initial adaptive refinement        = 0
+  set Initial global refinement          = 0
+  set Time steps between mesh refinement = 0
+end
+
+# Velocity on boundaries characterized by functions
+subsection Boundary velocity model
+  set Prescribed velocity boundary indicators = bottom y: function, top y: function, left x: function, right x: function
+  subsection Function
+    set Variable names      = x,y
+    set Function constants  = m=0.0005, year=1
+    set Function expression = if (x<50e3 , -1*m/year, 1*m/year); if (y<50e3 , 1*m/year, -1*m/year);
+  end
+end
+
+# Temperature boundary and initial conditions
+subsection Boundary temperature model
+  set Fixed temperature boundary indicators   = bottom, top, left, right
+  set List of model names = box
+  subsection Box
+    set Bottom temperature = 273
+    set Left temperature   = 273
+    set Right temperature  = 273
+    set Top temperature    = 273
+  end
+end
+subsection Initial temperature model
+  set Model name = function
+  subsection Function
+    set Function expression = 273
+  end
+end
+
+subsection Compositional fields
+  set Number of fields = 1
+  set Names of fields = right
+  set Compositional field methods = field
+end
+
+subsection Initial composition model
+  set List of model names = function
+  set List of model operators = add
+  subsection Function
+     set Coordinate system = cartesian
+     set Variable names = x, y
+     set Function constants = XMAX=100e3
+     set Function expression = (x>XMAX/2.0) ? 1.0 : 0.0
+  end
+end
+
+# Material model (values for background material)
+subsection Material model
+  set Model name = visco plastic
+  subsection Visco Plastic
+    set Phase transition depths = background:50e3|75e3, right:50e3|75e3
+    set Phase transition widths = background:1e3|1e3, right:1e3|1e3
+    set Phase transition temperatures = background:273|273, right:273|273
+    set Phase transition Clapeyron slopes = background:0|0, right:0|0
+
+    set Densities = background:3300|3400|3500, right:3600|3700|3800
+    set Thermal expansivities = background:3.5e-5|2e-5|1.5e-5, right:3.5e-5|2e-5|1.5e-5
+    set Heat capacities = background:1200|1000|900, right:1200|1000|900
+
+    # Set Maximum and Minimum viscosities on phases
+    set Maximum viscosity = background: 1.e21 | 1.e21 | 1.e21, right: 1.e20
```

Comment:
Please modify the values for one of the phases of the background field. Just to make sure it works. Then update the test results accordingly.